static/app/components/sidebar/index.spec.tsx

```diff
@@ -436,4 +420,39 @@ describe('Sidebar', function () {
       expect(await screen.findByTestId('floating-accordion')).toBeInTheDocument();
     });
   });
+
+  describe('Stacked navigation with navigation-sidebar-v2 flag', () => {
+    it('renders secondary subnav for issues', async function () {
+      renderSidebar({
+        organization: {...organization, features: ['navigation-sidebar-v2']},
+      });
+
+      expect(await screen.findByText('Issues')).toBeInTheDocument();
```

Comment:
Should it be `findByRole('link', {name: 'Issues'})`? Or is it a button `findByRole('button', {name: 'Issues'})`